src/main/java/cn/nukkit/entity/projectile/EntityThrownTrident.java

```diff
@@ -59,14 +59,14 @@ public EntityThrownTrident(FullChunk chunk, CompoundTag nbt, Entity shootingEnti
 
     public EntityThrownTrident(FullChunk chunk, CompoundTag nbt, Entity shootingEntity, boolean critical) {
         super(chunk, nbt, shootingEntity);
-        this.setCritical(critical);
     }
 
     @Override
     protected void initEntity() {
         super.initEntity();
 
         this.damage = namedTag.contains("damage") ? namedTag.getDouble("damage") : 8;
+        closeOnCollide = false;
```

Comment:
why you use false if false is default value? and this can spam 1000 entities on map. It's ok if you will make a "Event" "TridentEntityLaunchEvent" if anyone use them custom. Entity is not already implemented, this will create only lag.

Reply:
Sorry, I had set the default value incorrectly.